.vscode/settings.json

```diff
@@ -13,5 +13,6 @@
   "editor.formatOnType": false,
   "editor.formatOnSave": true,
   "editor.formatOnSaveMode": "file",
-  "files.autoSave": "onFocusChange"
+  "files.autoSave": "onFocusChange",
+  "cSpell.words": ["Placemark", "tinycolor", "UCSC"]
```

Comment:
👍 Didn't know we could actually add words to spellcheck here, that's cool! Maybe we could add some more commonly used terms in the future. Can't think of any atm.